CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 22.7.5
 
+### Breaking Changes
+- Flexible Privacy Groups support to Tessera's EC encryptor (contracts modified) [#4282](https://github.com/hyperledger/besu/pull/4282)
```

Comment:
can this be more specific about what the change is / why it's needed

Reply:
Just updated the text so it is more specific